python/tchannel/tornado/tchannel.py

```diff
@@ -103,15 +108,24 @@ def __init__(self, name, hostport=None, process_name=None,
         )
 
         self.name = name
+        self._trace = trace
 
         # register event hooks
         self.event_emitter = EventEmitter()
         self.hooks = EventRegistrar(self.event_emitter)
+        self.hooks.register(ZipkinTraceHook(tchannel=self))
```

Comment:
if the tracingflag set to False, the zipkin hook does nothing. In the trace session of protocol document, it says 
'Trace flags:

flag	description
0x01	tracing enabled for this request
When the enabled flag is not set, the tracing data is still required to be present in the frame, but the tracing data will not necessarily be sent to zipkin.' 
It implies the zipkin tracing is a built in feature.